semver/types.ts

```diff
@@ -49,8 +49,14 @@ export type FormatStyle =
 export interface SemVerComparator {
   operator: Operator;
   semver: SemVer;
-  min: SemVer;
-  max: SemVer;
+  /**
+   * @deprecated (will be removed in 0.212.0) use {@linkcode comparatorMin} instead.
+   */
+  min?: SemVer;
+  /**
+   * @deprecated (will be removed in 0.212.0) use {@linkcode comparatorMax} instead.
+   */
+  max?: SemVer;
```

Comment:
```suggestion
  min: SemVer;
  /**
   * @deprecated (will be removed in 0.212.0) use {@linkcode comparatorMax} instead.
   */
  max: SemVer;
```
Sorry, I just noticed these were made optional. I think that should be reverted.